app/experimenter/legacy-ui/rapid/components/visualization/PrimaryMetricTable.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+import { displaySelectOptionList } from "experimenter-rapid/components/experiments/utils";
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import {
+  PRIMARY_METRIC_COLUMNS,
+  TABLE_LABEL,
+  DISPLAY_TYPE,
+} from "experimenter-rapid/components/visualization/constants/analysis";
+import ResultsRow from "experimenter-rapid/components/visualization/ResultsRow";
+import { ExperimentData } from "experimenter-types/experiment";
+
+const getStatistics = (
+  probeset: string,
+): Array<{
+  name: string;
+  displayType: DISPLAY_TYPE;
+  branchComparison?: string;
+}> => {
+  const probesetMetricID = `${probeset}_ever_used`;
```

Comment:
Oh good yeah I wasn't sure what to think about the word `metric` here but `probsetMetric` makes a ton of sense thank you 🙏 🙏 🙏